server/views/pages/overview/partials/timelineTab/timelineTabContents.njk

```diff
@@ -7,17 +7,24 @@
       {% for event in timeline.events %}
         <div class="moj-timeline__item">
           <div class="moj-timeline__header">
-            <h2 class="moj-timeline__title">{{ event.eventType | formatTimelineEvent }} </h2>
+            <h2 class="moj-timeline__title">{{ event.eventType | formatTimelineEvent }}</h2>
             <p class="moj-timeline__byline govuk-!-display-none-print">by {{ event.actionedByDisplayName }}, {{ event.prison.prisonName if event.prison.prisonName else event.prison.prisonId }}</p>
           </div>
 
           <p class="moj-timeline__date">
             <time datetime="{{ event.timestamp }}">{{ event.timestamp | formatDate('D MMMM YYYY') }}</time>
           </p>
 
-          {% if event.eventType == 'ACTION_PLAN_CREATED' %}
+          {% if event.eventType == 'ACTION_PLAN_CREATED' or event.eventType == 'GOAL_UPDATED' %}
           <div class="moj-timeline__description">
-            <p class="govuk-body govuk-!-margin-bottom-0 govuk-!-display-none-print"><a href="/plan/{{ prisonerSummary.prisonNumber }}/view/overview">View {{ prisonerSummary.firstName }} {{ prisonerSummary.lastName | title }}'s learning and work progress</a></p>
+            <p class="govuk-body govuk-!-margin-bottom-0 govuk-!-display-none-print">
```

Comment:
This is OK for now, but once we add a 3rd type that has subtly different link/display requirements here this is going to get messy! 😢 

I wonder if the next time we need to touch this section of the template we look to do it a different way, perhaps with a conditional or "dynamic" include for each event type? 🤔 

As mentioned, its OK for now, but next time we need to touch this lets look at alternative ways of doing it 👍

Reply:
Yep makes sense, I think the "Goal created" event has the same link as "Goal updated" so we may be ok for a little longer - but any more differences then definitely.